README.md

```diff
@@ -23,3 +23,23 @@ An operator's CSV must contain the annotations mentioned [here](https://github.c
 ## Updating your OKD operator
 
 Similarly, to update your operator you need to submit a PR with any changes to your operator resources.
+
+## Pulling OperatorBundles from quay.io
+### Prerequists
+The following prequistis are required to the [quay.io](https://quay.io) scripts:
+
+* [jq](https://stedolan.github.io/jq/) must be installed
+* A [quay.io](https://quay.io) account
+
+To pull your operator bundle from quay.io, simple run the following command:
+```
+$ pull-from-quay $NAMESPACE $REPOSITORY
```

Comment:
`$ scripts/pull-from-quay $NAMESPACE $REPOSITORY`

Reply:
Resolved in the latest commit.